engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -641,6 +644,10 @@ void printUsage(String cmd)
         null,
         null,
         "Exit the CLI"));
+    globalCommands.put("quit", new CommandSpec(new ExitCommand(),
```

Comment:
Yes, alias is possible but I thought the alias is used by users to create their own aliases.